debian/rules

```diff
@@ -14,35 +14,36 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-INSTALL_DIR=debian/containerd.io
-CONTAINERD_BINARIES=bin/containerd bin/containerd-shim bin/ctr
-
 %:
 	dh $@ --with systemd
 
 # GO_SRC_PATH and PACKAGE are defined in the dockerfile
 # VERSION and REF are defined in scripts/build-deb
-bin/%: ## Create containerd binaries
-	@echo "+ make -C $(GO_SRC_PATH) --no-print-directory VERSION=$${VERSION} REVISION=$${REF} PACKAGE=$${PACKAGE} $@"
-	@make -C $(GO_SRC_PATH) --no-print-directory VERSION=$${VERSION} REVISION=$${REF} PACKAGE=$${PACKAGE} $@
-	mkdir -p $(@D)
-	mv -v $(GO_SRC_PATH)/$@ $@
+binaries: ## Create containerd binaries
+	@set -x; make -C $(GO_SRC_PATH) --no-print-directory \
+		DESTDIR="$$(pwd)" \
+		VERSION=$${VERSION} \
+		REVISION=$${REF} \
+		PACKAGE=$${PACKAGE} \
+		binaries install
+
+	# Remove containerd-stress, as we're not shipping it as part of the packages
+	rm -f bin/containerd-stress
```

Comment:
This one is weird, we should update `install` target upstream 🤔

Reply:
Yeah, was considering that, but likely wouldn't be back ported to existing release branches, so thought this was the "safer" option for now